src/recipients/RecipientItemV2.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { memo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ActivityIndicator, StyleSheet, Text, View } from 'react-native'
+import ContactCircle from 'src/components/ContactCircle'
+import Touchable from 'src/components/Touchable'
+import Logo, { LogoTypes } from 'src/icons/Logo'
+import QuestionIcon from 'src/icons/QuestionIcon'
+import { Recipient, getDisplayDetail, getDisplayName } from 'src/recipients/recipient'
+import colors, { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  recipient: Recipient
+  onSelectRecipient(recipient: Recipient): void
+  loading: boolean
+  showValoraIcon?: boolean
+  selected?: boolean
+}
+
+const ICON_SIZE = 10
+
+function RecipientItem({ recipient, onSelectRecipient, loading, showValoraIcon, selected }: Props) {
+  const { t } = useTranslation()
+
+  const onPress = () => {
+    onSelectRecipient(recipient)
+  }
+
+  return (
+    <Touchable onPress={onPress} testID="RecipientItem">
+      <View style={[styles.row, selected && styles.rowSelected]}>
+        <View>
+          <ContactCircle
+            style={styles.avatar}
+            recipient={recipient}
+            backgroundColor={Colors.gray1}
+            foregroundColor={Colors.dark}
+            borderColor={Colors.gray2}
+            DefaultIcon={() => <QuestionIcon />} // no need to honor color props here since the color we need match the defaults
+          />
+          {showValoraIcon && (
+            <Logo
+              type={LogoTypes.LIGHT}
+              style={styles.valoraIcon}
```

Comment:
type error here?

Reply:
yeah this was because the celo branding was not supporting this prop and CI uses it. Fixed when I updated the celo branding to accept this prop